cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -5,4 +5,11 @@ commonLabels:
 
 resources:
 - ../common
+- ../../bundles/external-secrets-clustersecretstore
```

Comment:
We're also missing ODF and vault as dependencies for externalsecrets on `nerc-ocp-prod`. @larsks would you prefer to go ahead and merge this despite the missing deps or wait until that's done?

Reply:
@larsks what are your thoughts on the next steps?

Reply:
@jtriley @joachimweyl we're not expecting to install vault on `nerc-ocp-prod`; we'll be using the vault instance on `nerc-ocp-infra`.

Reply:
@jtriley is this CR resolved after Lars' information?